src/style.css

```diff
@@ -1,3 +1,75 @@
-body {
+:root {
+  --primary-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 0.75rem;
+  font-weight: 500;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+}
+
+.navigation__list {
+  display: flex;
+  list-style-type: none;
+}
+
+.list__item {
+  text-transform: uppercase;
+}
+
+.list__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.list__link {
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+  color: #000;
+  font-weight: bold;
+}
+
+.list__link:hover {
+  color: var(--primary-color);
+  text-decoration: underline;
+  text-decoration-thickness: 2px;
+}
+
+.la__link {
+color: var(--primary-color);
+}
+
+.is-active {
+  color: var(--primary-color);
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  width: 37px;
```

Comment:
We never use broken numbers like this. This after should have 100% width, not a fixed value